src/transform.rs

```diff
@@ -11,10 +11,6 @@ use std::collections::HashMap;
 
 pub const DEFAULT_SPAWN_SIZE: f32 = 250.;
 
-pub type EdgeTransform<N, E, Ty, Ix, Dn, D> =
```

Comment:
It's a bit unfortunate that trait aliases are still unstable. Of course you could define a new trait and add blanket implementations, but this particular case has a lot of type bounds so the trait definition becomes very verbose. I opted to just use the `Fn` category of traits at the call sites instead.